packages/office-ui-fabric-react/src/components/Modal/examples/Modal.Basic.Example.tsx

```diff
@@ -21,7 +21,14 @@ export class ModalBasicExample extends React.Component<
     return (
       <div>
         <DefaultButton secondaryText="Opens the Sample Modal" onClick={this._showModal} text="Open Modal" />
-        <Modal isOpen={this.state.showModal} onDismiss={this._closeModal} isBlocking={false} containerClassName="ms-modalExample-container">
+        <Modal
+          titleAriaId="Sample Modal"
+          subtitleAriaId="Modal Dialog example"
```

Comment:
pretty sure this isn't what you intended... The `titleAriaId` and `subtitleAriaId` should be `id`s to elements, for example on the span that contains `'Lorem Ipsum'` on line 33 should have and `id`, like `title` and the `titleAriaId` would also be set to `title` (and a similar thing for the subtitle)